src/main/kotlin/org/rust/lang/core/resolve/Processors.kt

```diff
@@ -150,8 +150,10 @@ fun processAllWithSubst(
 
 fun filterCompletionVariantsByVisibility(processor: RsResolveProcessor, mod: RsMod): RsResolveProcessor {
     return fun(it: ScopeEntry): Boolean {
-        val visible = it.element as? RsVisible ?: return processor(it)
-        if (visible.isVisibleFrom(mod)) return processor(it)
-        return false
+        val element = it.element
+        if (element is RsVisible && !element.isVisibleFrom(mod)) return false
+        if (element is RsOuterAttributeOwner && element.queryAttributes.isDocHidden) return false
```

Comment:
Are you sure that we should filter such items if they are from our crate?

Reply:
Umm, yep, I remembered that in the case of macro we also check that it's in another module. I should add the same check here

Reply:
Fixed